src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -512,25 +513,14 @@ export default class ListPeopleAndRoles extends Mixins(CommonMixin, OrgPersonMix
   opacity: .4;
 }
 
-// Override Base Address font style/colour
-::v-deep {
-  // italicize the delivery instructions in the base address component
-  .address-block .delivery-instructions {
-    font-style: italic;
-  }
-
-  .address-block {
-    font-size: $px-14;
-    color: $gray7;
-  }
```

Comment:
These styles are now included in the BaseAddress common component, so these overrides are obsolete.